CMakeLists.txt

```diff
@@ -1,7 +1,10 @@
 # ubuntu 12.04 LTS cmake version 2.8.7
 # ubuntu 14.04 LTS cmake version 2.8.12.2
 # ubuntu 16.04 LTS cmake version 3.5.1
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5.1)
+set(CUDA_USE_STATIC_CUDA_RUNTIME OFF CACHE INTERNAL "" FORCE)
```

Comment:
I think this only needs to be called once, before FindCUDA.

Reply:
actually after :-)